docs/profiling.md

```diff
@@ -0,0 +1,121 @@
+# vHive profiling tool
+
+The tool focus on the performance characteristics of vHive framework. It uses a practical method, 
+[the TopDown method](https://ieeexplore.ieee.org/document/6844459), to identify potential 
+performance bottlenecks by colocating VMs on a single host. The tool issues requests to VMs in
+Round-Robin and takes metrics from the VMs, such as RPS per CPU, hardware counters, etc. If multiple
+groups of metrics are collected, then line charts can be drawn for each attribute of metrics. An example 
+is shown in the [section](###TestProfileIncrementConfiguration-function) below.
+
+## Methodology
+
```

Comment:
good that you started this section with enumerating the components. however, binding is not a component is just a part of the setup. It is supplementary.

Reply:
Done